packages/rest-crud/README.md

```diff
@@ -41,6 +41,19 @@ npm install --save @loopback/rest-crud
    app.controller(ProductController);
    ```
 
+## defineCrudRepositoryClass
+
+A helper method `defineCrudRepositoryClass` is exported by this module, which
+can be used for creating named repositories (based on the Model).
+
+Usage example:
+
+```ts
+const db = new juggler.DataSource({connector: 'memory'});
+const ProductRepository = defineCrudRepositoryClass(Product);
+const repo = new ProductRepository(db);
```

Comment:
Can you please show how to register the created repository class for dependency injection?

I think we need to:

1. Setup dependency injection by calling `inject` - see the instructions for the controller
2. Call `app.repository(ProductRepository)`

I think it would be better to merge this new content with the steps 1-5 above, perhaps rework the step 2 "Create a Repository class, e.g. using `lb4 repository` tool." to show how to use `defineCrudRepositoryClass`?

If you prefer to keep the instructions for the controller as they are now, then I am fine with that too, but please improve the overall structure, e.g. by introducing two subsections to `Basic use`, one for controllers and another for repositories.

Reply:
@bajtos I have added a new section **Integrated example**.